students/Jaidjen/lesson03/list_lab.py

```diff
@@ -0,0 +1,67 @@
+#Series 1
+
+Fruits=["Apples", "Pears", "Oranges", "Peaches"]
+print(Fruits)
+
+New_Fruit=input('Please add another fruit to the list: ')
```

Comment:
pep8 style violation: var names (and function names) should use snake_case -> `new_fruit`